### DISCUSSION TITLE
Space-amoebas flocking and using truck horn to clear

### DISCUSSION THREAD

**Discussion by jelster** [jelster/space-truckers#27] [Ideas]
Flocks of space amoebas prowl the space ways, so,stoke interfering with traffic. Seasoned operators know to blow their trucks' space-air horn to clear the space-highway

**Comment by jelster** [comment]
My thought is that this would be a nice way to show off both thin instances AND newly added support for infinite (effectively) morph targets. Working on some sketches

**Reply by eldinor** [reply]
Morphing can be perf heavy, so take care :)

**Reply by jelster** [reply]
Definitely will need to test performance impact, but with instances, aren't you only paying once for the mesh and the morph targets? New way of storing morph target data in the shaders is to put the extra vertice data into a lookup texture, which might help with memory usage

**Comment by jelster** [comment]
Here's a quick sketch showing the space amoeba in different stages as it moves ![image](https://user-images.githubusercontent.com/1508500/106400636-3f901580-63e5-11eb-86e8-37d5e9598766.png)
